plugins/check_apt_upgrade

```diff
@@ -113,7 +120,7 @@ fi
 # Note that we use `"$@"' to let each command-line parameter expand to a
 # separate word. The quotes around `$@' are essential!
 # We need TEMP as the `eval set --' would nuke the return value of getopt.
-TEMP=$($GETOPT -q -s sh -n "$SCRIPT_NAME" -o +hnvt: -l skip-hwe-check -- "$@")
+TEMP=$($GETOPT -q -s sh -n "$SCRIPT_NAME" -o +hlnvt: -l skip-hwe-check -- "$@")
```

Comment:
```suggestion
TEMP=$($GETOPT -q -n "$SCRIPT_NAME" -o +hlnvt: -l skip-hwe-check -- "$@")
```
WDYT? `man 1 getopt` says `-s bash` is the default.

Reply:
I agree, but this is from your code. I will change this and upload with the merged shellcheck fixes

Reply:
Right but since you changed the interpreter it makes sense to change it now, I think

Reply:
This was a misstake, I changed back to sh and readded the -s sh now